### ISSUE TITLE
Localized to the lower eyelid

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11046]
Preferred term label:
Localized to the lower eyelid

Synonyms
Restricted to the lower eyelid, Affecting the lower eyelid, Lower eyelid involvement

Definition (free text, please give PubMed ID)
A descriptor indicating that a condition, lesion, or abnormality is confined to the lower eyelid. This localization may be relevant for various ocular conditions, including infections, structural abnormalities, or dermatological diseases affecting the lower eyelid region.

Parent term (use hpo.jax.org/app)
Localized (HP:0012838)

Diseases characterized by this term
Lower eyelid entropion , Chalazion, Basal cell carcinoma of the eyelid

Your nano-attribution (ORCID)
0000-0001-9042-0508

ADDITIONALLY:
We also need a parent term "Localized to the eyelids" grouping this new one and "Localized to upper eyelid "[HP:6000059].

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001265
- label: Localized to the lower eyelid
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: A descriptor indicating that a condition, lesion, or abnormality is confined to the lower eyelid. This localization may be relevant for various ocular conditions, including infections, structural abnormalities, or dermatological diseases affecting the lower eyelid region.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11046
- synonyms:
